docs/concepts/Alexa-Hosted-Skill-Commands.md

```diff
@@ -102,11 +102,26 @@ Unlike in CLI v1, running `ask deploy` on hosted skills will no longer trigger s
 Instead, `$ git push` sends the latest source code to Lambda.  It also deploys skill-package changes such as the interaction model, skill manifest and in-skill products.
 
 ## DEPLOYMENT STEPS:
-* Push to deploy skill code and reosurces:
+* Push to deploy skill code and resources:
 	* `$ git push`
-  * Pushing in `master` branch deploys "lambda" folder to user's `development` stage AWS Lambda function, and deploys "skill-package" folder as skill's JSON files
-  * Pushing in `prod` branch deploys "lambda" folder to user's `live` stage AWS Lambda function, and deploys "skill-package" folder as skill's JSON files
-
+  * Pushing to `master` branch deploys "lambda" folder to user's `development` stage AWS Lambda function, and deploys "skill-package" folder as skill's JSON files
+  * Pushing to `prod` branch deploys "lambda" folder to user's `live` stage AWS Lambda function, and deploys "skill-package" folder as skill's JSON files
+  * Pushing to `dev` branch updates "lambda" folder displayed in the web console. You only need to push to dev branch if you want to sync version that is displayed in the web console and receiving the following warning: "The master branch of your skill is ahead of the dev version. To re-enable the code editor, you need to use the ASK CLI to merge the remote master branch to dev and push."
```

Comment:
I think we need to treat it differently then the intro to prod and master branches. Maybe 
```
Note: `dev` branch is only used to be displayed in the web console. You only need to push to dev branch if you want to sync version that is displayed in the web console. 

When you receive the following warning: "The master branch of your skill is ahead of the dev version. To re-enable the code editor, you need to use the ASK CLI to merge the remote master branch to dev and push.", you need to use following steps to sync your master branch back to dev:
... (the steps to fix)
```